src/Catalyst.Core.Modules.Dfs.Tests/IntegrationTests/DagNodeTest.cs

```diff
@@ -0,0 +1,309 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System.IO;
+using System.Linq;
+using System.Text;
+using Catalyst.Abstractions.Dfs;
+using Catalyst.Core.Lib.Dag;
+using Catalyst.Core.Modules.Hashing;
+using Google.Protobuf;
+using MultiFormats;
+using MultiFormats.Registry;
+using Xunit;
+
+namespace Catalyst.Core.Modules.Dfs.Tests.IntegrationTests
+{
+    public sealed class DagNodeTest
+    {
+        public DagNodeTest()
+        {
+            new HashProvider(HashingAlgorithm.GetAlgorithmMetadata("sha2-256"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove this useless object instantiation of class 'HashProvider' or use it.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)